### PR TITLE
Enable linux aarch64 build

### PR DESCRIPTION
**Pull request by artanokhov**

### CODE REVIEW

**Comment by artanokhov** [comment]
@conda-forge-admin please rerender

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by artanokhov** [comment]
@conda-forge-admin please rerender

File: recipe/patches/002-aarch64-gapi-narrowing-conv.patch [hunk 1]
```diff
@@ -0,0 +1,26 @@
+diff --git a/thirdparty/ocv/opencv_hal_neon.hpp b/thirdparty/ocv/opencv_hal_neon.hpp
+index 83d561d211..af5e788569 100644
+--- a/thirdparty/ocv/opencv_hal_neon.hpp
++++ b/thirdparty/ocv/opencv_hal_neon.hpp
+@@ -2563,14 +2563,17 @@ CV_ALWAYS_INLINE void v_deinterleave_expand(const v_uint8x16& src,
+                                             v_int16x8& even, v_int16x8& odd)
+ {
+     constexpr int nlanes = static_cast<int>(v_uint8x16::nlanes);
+-    uchar mask_e[nlanes] = { 0, -1, 2, -1, 4, -1, 6, -1,
```
Comment (ilya-lavrenov) [inline recipe/patches/002-aarch64-gapi-narrowing-conv.patch:9]: I suppose we can use `0xff` instead of `-1`

File: conda-forge.yml [hunk 2]
```diff
@@ -5,8 +5,11 @@ azure:
       vmImage: windows-2019
 build_platform:
   osx_arm64: osx_64
+  linux_aarch64: linux_64
 os_version:
   linux_64: cos7
+provider:
+  linux_aarch64: azure
```
Comment (xhochy) [inline conda-forge.yml:12]: If you already set `build_platform`, setting `provider` has no effect.
```suggestion
```

Comment (artanokhov) [reply conda-forge.yml]: Thank you, I've just took opencv-feedstock as reference for it :)
https://github.com/conda-forge/opencv-feedstock/blob/main/conda-forge.yml

**Comment by ilya-lavrenov** [comment]
Newer release is out https://github.com/conda-forge/openvino-feedstock/pull/2

**Pull request closed without merging**